etc/helm/pachyderm/envoy.json

```diff
@@ -0,0 +1,1027 @@
+{
```

Comment:
what's the process a dev should go through to updating this file when changing the jsonnet?

Reply:
"make" in this directory.  If you forget, CI will remind you.

(So... why not put a comment in this file saying that it's autogenerated.  Why not use YAML instead of JSON?  jsonnet didn't have a way to pass a comment at the very top level through 😢.)